utils/common_device_api.c

```diff
@@ -151,29 +151,15 @@ size_t GetModelNum( char *pModelNum, size_t szBufSize )
 {
     size_t i = 0;
     FILE *fp;
-    char *pTmp;
-    char buf[150];
 
     if( pModelNum != NULL )
     {
         *pModelNum = 0;
-        if( (fp = fopen( DEVICE_PROPERTIES_FILE, "r" )) != NULL )
+        if( (fp = fopen( "/tmp/.model_number", "r" )) != NULL )
```

Comment:
Which component will update model number inside /tmp/.model_number file.
Let us consider if the tmp/.model_number file does not have any data then how to get model number?

Reply:
Removed this logic